### PR TITLE
prepare v0.3.0 release

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

File: bundle/triggermesh-operator.v0.3.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -57,18 +58,18 @@ spec:
...
+
     ### Supported Features:
     **Knative services**
     Knative services can be deployed from source manifest or by specifying a container image or by specifying a git repository.
     **Knative Build**
     Container builds and more generically workflows can be configured and deployed.
     **Knative Eventing**
     Event sources such as GitHub, GitLab and a few AWS event sources can be configured and deployed.
-  displayName: Operator
+  displayName: Triggermesh
```
Comment (sameersbn) [inline bundle/triggermesh-operator.v0.3.0.clusterserviceversion.yaml:72]: There exists another operator named `Triggermesh` in the operator cloud. We need to unpublish that one for this change

File: bundle/triggermesh-operator.v0.3.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -225,7 +226,7 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: triggermesh-operator
-                image: gcr.io/triggermesh/tmoperator:v0.1.0
+                image: gcr.io/triggermesh/operator:v0.3.0
```
Comment (sameersbn) [inline bundle/triggermesh-operator.v0.3.0.clusterserviceversion.yaml:229]: confused about this part. We're "publishing" images are redhat but using images from GCR.

File: helm-charts/triggermesh/values.yaml [hunk 3]
```diff
@@ -17,13 +17,13 @@ knative:
   manageConfigDomain: false
 
 frontend:
-  baseHost: cloud.tm.demo.vshn.net
+  baseHost: cloud.apps.tm.clusters.openshiftcorp.com
   authDomain: triggermesh.auth0.com
   authApiId: # Auth0 ID
   sentryDSN: # https://<id>@sentry.io/id
   replicaCount: 3
   image: triggermesh/tmfront
```
Comment (sameersbn) [inline helm-charts/triggermesh/values.yaml:25]: same with these images, they're coming from gcr.io and are hosted on the public instance

File: bundle/triggermesh-operator.v0.3.0.clusterserviceversion.yaml [hunk 4]
```diff
@@ -8,46 +8,47 @@ metadata:
...
                 "enabled": true
-              }
+              },
             },
-            "frontend": {
-              "auth0ClientId": "\u003cREDACTED\u003e",
-              "authApiId": "\u003cREDACTED\u003e",
-              "baseHost": "cloud-test.tm.demo.vshn.net",
+            "backend": {
+              "baseHost": "api.apps.triggermesh.clusters.openshiftcorp.com",
```
Comment (sameersbn) [inline bundle/triggermesh-operator.v0.3.0.clusterserviceversion.yaml:27]: used URL's to match the README

File: bundle/triggermesh-operator.v0.3.0.clusterserviceversion.yaml (hunk 1 above)
Comment (sebgoa) [reply bundle/triggermesh-operator.v0.3.0.clusterserviceversion.yaml:72]: TriggerMesh ?

File: bundle/triggermesh-operator.v0.3.0.clusterserviceversion.yaml (hunk 2 above)
Comment (sebgoa) [reply bundle/triggermesh-operator.v0.3.0.clusterserviceversion.yaml:229]: yes, that's doable

Comment (sebgoa) [reply bundle/triggermesh-operator.v0.3.0.clusterserviceversion.yaml:229]: yes that's normal

**Pull request merged**